SU-SWS/stanford_magazine
# SOE-2354: Stanford Magazine feature revert

# READY FOR REVIEW

# Summary
- This adds the help text for adding audio to a magazine article
- This adds the _list_ dependency to the feature
- Feature excludes the XML site map variable
- Adds _update_7300()_  to set the default featured image for magazine articles
- Removes the strong arm variable _stanford_magazine_article_image_ since it's getting added by _update_7300()_
- Updates the default image to match what is on _Prod_

# Needed By (Date)
- Sprint end

# Criticality
- Client is eagerly awaiting this.

# Steps to Test
- Switch to this branch ($ git checkout 7.x-1.x-bookend3)
- revert _Stanford Magazine_
- Clear cache
- Navigate to node/add/stanford-magazine-article 
- Verify you see: To override the fixed height add class="iframe-auto" after the iframe tag

See:

- Ticket: https://stanfordits.atlassian.net/browse/SOE-2354
- https://stanfordits.atlassian.net/secure/attachment/37330/Screen%20Shot%202017-10-30%20at%2010.18.59.png
- https://stanfordits.atlassian.net/secure/attachment/37331/Screen%20Shot%202017-10-30%20at%2010.23.26.png
- https://stanfordits.atlassian.net/secure/attachment/37332/Screen%20Shot%202017-10-30%20at%2010.23.02.png


Fix for the default image:
- Run update.php or navigate to: /update.php
- Clear cache
- Navigate to /admin/content/default-image to verify


`Default image for Content > Magazine article > Featured Image
Image icon stanford_magazine_article_image.png Remove
Variable name: stanford_magazine_article_image.`

See: 

- Ticket: https://stanfordits.atlassian.net/browse/SOE-2354
- https://stanfordits.atlassian.net/secure/attachment/37333/Screen%20Shot%202017-10-30%20at%2010.25.27.png
- https://stanfordits.atlassian.net/secure/attachment/37334/Screen%20Shot%202017-10-30%20at%2010.31.44.png

# Affects 
- SoE (School of Engineering website)

# Associated Issues and/or People
## Related JIRA ticket(s)
https://stanfordits.atlassian.net/browse/SOE-2354

## Related PRs

## More Information

## Folks to notify
@kbrownell 


# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
